extensions/stackdriver/config/v1alpha1/stackdriver_plugin_config.proto

```diff
@@ -80,7 +80,7 @@ message PluginConfig {
 
   // Optional. Controls whether or not to export mesh edges to a mesh edges
   // service. This is disabled by default.
-  bool enable_mesh_edges_reporting = 3;
+  bool enable_mesh_edges_reporting = 3 [deprecated = true];
```

Comment:
should we update the comment here as well. Something to the effect of `Deprecated -- Mesh edge reporting is no longer supported` ?

Reply:
Done.

Reply:
Should it be stronger than deprecated,  “This setting has no effect”

Reply:
Updated to make it more clear that this option is no-op.